### ISSUE TITLE
Update Readme with the need of installing libmariadbclient-dev on ubuntu

### ISSUE THREAD

**Issue by antolinos** [ispyb/py-ispyb#90]
At least in Ubuntu distributions the command `sudo pip install -r requirements.txt ` will fail if libmariadbclient-dev is not installed with this message:
```
    ERROR: Command errored out with exit status 1:
     command: /home/ale/miniconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-nhuytjq7/mysqlclient/setup.py'"'"'; __file__='"'"'/tmp/pip-install-nhuytjq7/mysqlclient/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-gpaelu57
         cwd: /tmp/pip-install-nhuytjq7/mysqlclient/
    Complete output (12 lines):
    /bin/sh: 1: mysql_config: not found
    /bin/sh: 1: mariadb_config: not found
    /bin/sh: 1: mysql_config: not found
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-nhuytjq7/mysqlclient/setup.py", line 15, in <module>
        metadata, options = get_config()
      File "/tmp/pip-install-nhuytjq7/mysqlclient/setup_posix.py", line 65, in get_config
        libs = mysql_config("libs")
      File "/tmp/pip-install-nhuytjq7/mysqlclient/setup_posix.py", line 31, in mysql_config
        raise OSError("{} not found".format(_mysql_config_path))
    OSError: mysql_config not found
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

```

In order to prevent this error someone could:
```
apt-get install libmariadbclient-dev
```
I wonder if it would be worth to mention in the Readme page

**Comment by IvarsKarpics** [comment]
I faced the issue as well and forgot to mention it in the readme. I will open a pr to fix this. BTW for travis I used sudo apt-get install -y python3-mysqldb: https://github.com/ispyb/py-ispyb/blob/master/.travis.yml

**Comment by antolinos** [comment]
I can do that if you want. I can give also some instructions to set up the database. Stuff like how to create the mxuser, grant privileges to the database, etc... it is simple but it might save time to people that want to jump in quickly

**Comment by IvarsKarpics** [comment]
I have a script to do that: https://github.com/ispyb/py-ispyb/blob/master/scripts/create_core_db.sh and again: travis is using it for testing. I guess there should be more info for developers...

**Comment by IvarsKarpics** [comment]
Resolved via #91
